SharedCode/src/commonMain/kotlin/StationsHelper.kt

```diff
@@ -0,0 +1,16 @@
+package com.jetbrains.handson.mpp.mobile
+
+class StationsHelper {
+    public val stations = arrayOf<Station>(
```

Comment:
Presumably this is temporary before you grab the stations from the API?

Reply:
Yes :)